src/erfaversion.c

```diff
@@ -0,0 +1,7 @@
+#include <config.h>
+
+const char* erfa_version() {
```

Comment:
Also, it can be useful to make the version integers available so that you can use them in your code without having to parse the string. See what I did in PAL (closely related to SOFA): https://github.com/Starlink/pal/blob/master/palVers.c